firebase-admin 13 is not supported on adapter/firebase

### Adapter type

@auth/firebase-adapter

### Environment

```
System:
    OS: macOS 15.6.1
    CPU: (8) arm64 Apple M2
    Memory: 160.75 MB / 24.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 24.9.0 - /opt/homebrew/bin/node
    Yarn: 1.22.22 - /opt/homebrew/bin/yarn
    npm: 11.6.0 - /opt/homebrew/bin/npm
    pnpm: 10.15.0 - ~/Library/pnpm/pnpm
    Deno: 2.5.2 - /opt/homebrew/bin/deno
  Browsers:
    Chrome: 140.0.7339.208
  npmPackages:
    @auth/firebase-adapter: ^2.10.0 => 2.10.0 
    next: ^15.5.0 => 15.5.0 
    next-auth: 5.0.0-beta.25 => 5.0.0-beta.25 
    react: 18.3.1 => 18.3.1
```

### Reproduction URL

https://github.com/not-needed

### Describe the issue

Adapter firebase need to be updated to firebase-admin 13+.

Errors:
```
npm install
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @auth/firebase-adapter@2.10.0
npm error Found: firebase-admin@13.4.0
npm error node_modules/firebase-admin
npm error   firebase-admin@"^13.4.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer firebase-admin@"^12.0.0" from @auth/firebase-adapter@2.10.0
npm error node_modules/@auth/firebase-adapter
npm error   @auth/firebase-adapter@"^2.10.0" from the root project
npm error
npm error Conflicting peer dependency: firebase-admin@12.7.0
npm error node_modules/firebase-admin
npm error   peer firebase-admin@"^12.0.0" from @auth/firebase-adapter@2.10.0
npm error   node_modules/@auth/firebase-adapter
npm error     @auth/firebase-adapter@"^2.10.0" from the root project
```

### How to reproduce

Make @auth/firebase-adapter a dependency on a project using firebase-admin@13.4.0.

### Expected behavior

To allow usage with firebase-admin@13+

--- a/packages/adapter-firebase/package.json
+++ b/packages/adapter-firebase/package.json
@@ -45,7 +45,7 @@
     "@auth/core": "workspace:*"
   },
   "peerDependencies": {
-    "firebase-admin": "^12.0.0 | ^13.0.0"
+    "firebase-admin": "^12.0.0 || ^13.0.0"
   },
   "devDependencies": {
     "firebase-admin": "^13.5.0"

--- a/packages/adapter-firebase/package.json
+++ b/packages/adapter-firebase/package.json
@@ -45,9 +45,9 @@
     "@auth/core": "workspace:*"
   },
   "peerDependencies": {
-    "firebase-admin": "^12.0.0"
+    "firebase-admin": "^12.0.0 | ^13.0.0"
   },
   "devDependencies": {
-    "firebase-admin": "^12.0.0"
+    "firebase-admin": "^13.5.0"
   }
 }

